packages/rocketchat-ui-login/client/login/form.html

```diff
@@ -85,7 +85,7 @@ <h2 data-i18n="Registration_Succeeded">{{{_ "Registration_Succeeded"}}}</h2>
 									<div class="rc-input__wrapper">
 										<input name="confirm-pass" id="confirm-pass" type="password" class="rc-input__element"
 											autocapitalize="off" autocorrect="off"
-											placeholder="{{passwordPlaceholder}}" autofocus>
+											placeholder="Confirm Password" autofocus>
```

Comment:
Please do not remove any i18n and instead add an additional place. And please remove the `package-lock.json` file changes.

Reply:
sorry, but I'm new in open Source. Could you please describe what do you mean by add an additional place and is my approach correct

Reply:
Replace “Confirm Password” with a placeholder and add a translation to the i18n file.

Reply:
should I add the same for confirm password  here in the form.js file in same folder. on the other hand there are many i18n files which one to use

Reply:
Forget my last comment.

There's a lot more to do, as `passwordPlaceholder` is not an i18n placeholder, it's a thru the admin ui configurable variable. Best thing to do would be an extra `confirmPasswordPlaceholder` option, configurable thru the admin ui. In addition you could add a default value (i18n translated) which will be shown, when there's no placeholder text set.

Search for "passwordPlaceholder" in your editor to get a good start:
![screen shot 2018-03-02 at 19 32 09](https://user-images.githubusercontent.com/4711233/36915410-d5669ef4-1e50-11e8-9758-30f0d11f4ea4.png)